src/daft-sql/src/planner.rs

```diff
@@ -686,7 +694,28 @@ impl SQLPlanner {
             SQLExpr::IsNotDistinctFrom(_, _) => {
                 unsupported_sql_err!("IS NOT DISTINCT FROM")
             }
-            SQLExpr::InList { .. } => unsupported_sql_err!("IN LIST"),
+            SQLExpr::InList {
+                expr,
+                list,
+                negated,
+            } => {
+                let expr = self.plan_expr(expr)?;
+                let list = list
+                    .iter()
+                    .map(|e| self.plan_lit(e))
+                    .collect::<SQLPlannerResult<Vec<_>>>()?;
+                // We should really have a better way to use `is_in` instead of all of this extra wrapping of the values
+                let series = literals_to_series(&list)?;
+                let series_lit = LiteralValue::Series(series);
+                let series_expr = Expr::Literal(series_lit);
+                let series_expr_arc = Arc::new(series_expr);
+                let expr = expr.is_in(series_expr_arc);
+                if *negated {
```

Comment:
this seems to be repeated a bunch. don't know if there is a way to make this simpler `negate_if` ? or if this is just a bit too much. I could see accidentally missing `.not` in one of these. unsure though feel free to do nothing.